src/screens/CreateCommunity/index.js

```diff
@@ -97,10 +97,9 @@ const CreateCommunity = () => {
     if (name && name.length >= MINIMUM_NAME_LENGTH && typeFetch === 'available') {
       const response = await submitCommunityName(name);
       if (response.success) {
-        navigation.replace('ContactScreen', {
+        navigation.replace('CreateCommunityCustomize', {
           isCreateCommunity: true,
-          topicCommunityId: response.topic_id,
-          topicCommunityName: name
+          topic: name
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The navigation destination has been changed from 'ContactScreen' to 'CreateCommunityCustomize'. Ensure that the 'CreateCommunityCustomize' screen is properly set up to handle the props being passed to it (`isCreateCommunity` and `topic`). Also, make sure that any functionality or state previously dependent on `topicCommunityId` and `topicCommunityName` in the 'ContactScreen' is appropriately handled in the 'CreateCommunityCustomize' screen.
```diff
-        navigation.replace('ContactScreen', {
-           isCreateCommunity: true,
-          topicCommunityId: response.topic_id,
-          topicCommunityName: name
-         });
+        navigation.replace('CreateCommunityCustomize', {
+           isCreateCommunity: true,
+           topic: name
+         });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->